mypy.ini

```diff
@@ -0,0 +1,25 @@
+[mypy]
+
+files = torchmultimodal
+
+warn_unused_configs = True
+warn_redundant_casts = True
+show_error_codes = True
+show_column_numbers = True
+check_untyped_defs = True
+pretty = True
+strict_optional = False
+allow_redefinition = True
+namespace_packages = True
+install_types = True
+
+exclude = models/flava.py|modules/losses/flava.py
+
+[mypy-PIL.*]
+ignore_missing_imports = True
```

Comment:
Is this because pillow is installed with torchvision?
Wonder if we only need this for pytorch domain libs?